opencog/atomspace/AtomSpace.h

```diff
@@ -673,22 +703,50 @@ class AtomSpace
     /** DEPRECATED! Do NOT USE IN NEW CODE!
```

Comment:
because code will get simpler and more readable if you just "do it yourself" instead of calling these wrapper ffunctions.